ansible/environments.yml

```diff
@@ -62,6 +62,7 @@
       tags:
         - uat
         - os
+        - os-uat
```

Comment:
This seems inconsistent with tags in other envs? Where is this used? Why is it needed?

Reply:
Yes, it looks inconsistent. The problem it solves is that ansible cannot "combine" tags. Without this tag we can't run tasks for UAT on OS only. We don't have other environments on AWS, thus the inconsistency.